Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-4.2.xml

```diff
@@ -0,0 +1,541 @@
+        <ns4:element name="activeDatetime" type="System.DateTime"/>
+        <ns4:element name="startDatetime" type="System.DateTime"/>
+        <ns4:element name="stopDatetime" type="System.DateTime"/>
+        <ns4:element name="negationRationale" type="System.Concept"/>
+        <ns4:element name="refills" type="System.Integer"/>
+        <ns4:element name="dose" type="System.Quantity"/>
+        <ns4:element name="frequency" type="System.Concept"/>
+        <ns4:element name="route" type="System.Concept"/>
+        <ns4:element name="method" type="System.Concept"/>
+        <ns4:element name="reason" type="System.Concept"/>
+        <ns4:element name="cumaltiveMedicationDuration" type="System.Quantity"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.PatientCharcteristicBirthdate" baseType="System.Any"
+                  retrievable="true" label="Patient Characteristic Birthdate" identifier="PatientCharcteristicBirthdate" primaryCodePath="code">
+        <ns4:element name="startDatetime" type="System.DateTime"/>
+        <ns4:element name="stopDatetime" type="System.DateTime"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.SubstanceRecommended" baseType="QDM.QDMBaseType">
+        <ns4:element name="startDatetime" type="System.DateTime"/>
+        <ns4:element name="stopDatetime" type="System.DateTime"/>
+        <ns4:element name="negationRationale" type="System.Concept"/>
+        <ns4:element name="reason" type="System.Concept"/>
+        <ns4:element name="dose" type="System.Quantity"/>
+        <ns4:element name="frequency" type="System.Concept"/>
+        <ns4:element name="method" type="System.Concept"/>
+        <ns4:element name="refills" type="System.Concept"/>
+        <ns4:element name="route" type="System.Concept"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.InterventionRecommended" baseType="QDM.QDMBaseType">
+        <ns4:element name="startDatetime" type="System.DateTime"/>
+        <ns4:element name="stopDatetime" type="System.DateTime"/>
+        <ns4:element name="negationRationale" type="System.Concept"/>
+        <ns4:element name="reason" type="System.Concept"/>
+    </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.PhysicalExamPerformed" baseType="QDM.QDMBaseType"
+                  retrievable="true" label="Physical Exam, Performed" identifier="PhysicalExamPerformed" primaryCodePath="code">
+        <ns4:element name="startDatetime" type="System.DateTime"/>
+        <ns4:element name="stopDatetime" type="System.DateTime"/>
+        <ns4:element name="negationRationale" type="System.Concept"/>
+        <ns4:element name="reason" type="System.Concept"/>
+        <ns4:element name="method" type="System.Concept"/>
+        <ns4:element name="result" type="System.Concept"/>
+        <ns4:element name="quantitativeResult" type="System.Quantity"/>
```

Comment:
OK, cool.  I see that you support both the concept and quantity versions of `result` here.  Personally, my preference would be to align more with how FHIR differentiates typed properties (via a suffix), so... `resultCode` (or `resultConcept`) and `resultQuantity`.